src/cache/cache.rs

```diff
@@ -125,7 +125,19 @@ impl CacheWrite {
     pub fn put_object<T>(&mut self, name: &str, from: &mut T, mode: Option<u32>) -> Result<()>
         where T: Read,
     {
-        let opts = FileOptions::default().compression_method(CompressionMethod::Deflated);
+        let method = if let Ok(ref method) = env::var("SCCACHE_COMPRESSION_METHOD") {
+            match method.to_lowercase().trim() {
+                "deflate" => CompressionMethod::Deflated,
+                "none" => CompressionMethod::Stored,
+                _ => {
+                    warn!("Unknown compression method {} using 'deflate'.", method);
```

Comment:
Sorry for being imprecise. The default logging setup doesn't show any logs. Furthermore until #115 is merged it's hard for the client to get the logs at all. On a CI system you also might have the case that it's unclear which specific job the starts the server. Restarting to ensure log settings might break the one or other job running in parallel. Personally I'd prefer to see cache errors in the stats if I did something wrong in the cache configuration.